src/api/guardian.js

```diff
@@ -0,0 +1,37 @@
+const Guardian = require("guardian-js");
```

Comment:
also this is 37 lines of code came out of no where !

Reply:
if you check the master these 37 lines are already there. It's not new